Model/Config.php

```diff
@@ -207,11 +229,7 @@ public function isCurrentStoreCurrencyUSD()
     {
         $currentCurrency = $this->storeManager->getStore()
             ->getCurrentCurrencyCode();
-        $isUSD = true;
-        if ($currentCurrency != self::CURRENCY_CODE) {
-            $isUSD = false;
-        }
-        return $isUSD;
+        return $currentCurrency == self::CURRENCY_CODE_USD;
```

Comment:
This just returns USD, do we need to add other currencies?

Reply:
This function checks if the currency is USD and returns a bool